src/adapters/store.js

```diff
@@ -2,112 +2,170 @@ import nconf from '@qwant/nconf-getter'
 import Error from '../adapters/error'
 import {version} from '../../config/constants.yml'
 import ExtendedString from "../libs/string";
+import LocalStore from "../libs/local_store"
+import MasqStore from "../libs/masq"
 
-let moduleConfig = nconf.get().store
+export default class Store {
 
-const AbStore = require(`../libs/${moduleConfig.name}`)
-const abstractStore = new AbStore(moduleConfig.endpoint)
+  constructor() {
+    // get store from window if already initialized
+    if (window.__store) {
```

Comment:
I have gather more data about our "problem" 
https://stackoverflow.com/questions/519520/difference-between-static-class-and-singleton-pattern
the fact that this class is never used as instance made me think that we should declare it as a "static class"
now it's a little late